documentation/airport-format.md

```diff
@@ -1,179 +1,168 @@
+          "28R": [["SENZY", "A25+"], "ZUPAX"]           // symbol ('|'). (ex: ["FIXNAME", "A50-|S220+"]) ("A50-" = 5,000ft or less, "S220+" = 220kts or more)
       },
       "draw": [["ENI*","PYE"], ["MXW*","PYE"], ["PYE","STINS","HADLY","OSI"]]
-        // (req) This "draw" section is what defines how the SID is to be drawn on the scope.
-        // The array contains multiple arrays that are a series of points to draw fixes between.
-        // In this case, ENI->PYE, MXW->PYE, PYE->STINS->HADLY->OSI are the lines drawn.
-        // Additionally, you'll notice two asterisks ('*'). This is an optional flag that, if invoked for "FIXXX"
-        // will tell canvas.js to write "FIXXX.PYE1" next to FIXXX on the scope. If no such flags are present,
-        // then the ICAO identifier for the SID will be drawn at the first point of the "draw" array. For european-
-        // style STARs, where they always end at the fix for which the STAR is named, don't use the flags.
+      // This section is what defines how the STAR is drawn on the map. The array contains multiple arrays that
+      // are a series of fixes to draw lines between. In this case, ENI->PYE, MXW->PYE, PYE->STINS->HADLY->OSI
+      // are the lines drawn. Additionally, you'll notice two asterisks ('*'). This is an optional flag that,
+      // if invoked for "FIXXX", will tell canvas.js to write "PYE1.FIXXX" next to "FIXXX" on the map. If no
+      // such flags are present, then the ICAO identifier for the SID will be drawn at the last point of the
+      // "draw" array. For European-style SIDs, where they always end at the fix for which the SID is named,
+      // don't use the flags. But if your SID has transitions, like in the N/S Americas, United Kingdom, etc.,
+      // be sure to flag all the transition fixes.
     }
   },
-  "departures": {
-    "airlines": [
-      ["three-letter ICAO airline code/fleet", 0], // see "Aircraft/Airline selectors" below
-      ...
-    ],
-    "destinations": [
-      "LISST", "OF", "SIDS", "ACRFT", "WILLL", "FLYYY", "TO"  // these must each be a defined SID above
-    ],
-    "type": ,
-    "offset": ,
-    "frequency": [3, 4] // the frequency, in minutes, of a new departing aircraft. A random number is chosen between the two.
+  "spawnPatterns": [
+  {
+      "origin": "KSFO",                     // Four-letter ICAO identifier of the origin point/fix. (can be left empty if "destination" is filled)
+      "destination": "vhhh",                // Four-letter ICAO identifier of the destination point/fix. (can be left empty if "origin" is filled)
+      "category": "arrival",                // Category of the flight. (arrival or departure, usually)
+      "route": "ABBEY..MUSEL..TAMAR..TD",   // Route of the flight. (can use fixes OR SIDs / STARs in this format: *origin*.*SID/STAR ICAO*.*destination*)
+      "altitude": 10000,                    // Altitude of the flight. (in feet)
+      "speed": 250,          
+      "method": "surge",
```

Comment:
"random" should be used for a realistic traffic pattern according to @erikquinn :+1: